e2e-tests/src/main/java/io/kafbat/ui/screens/panels/NaviSideBar.java

```diff
@@ -45,7 +46,7 @@ public String getPagePath(MenuItem menuItem) {
   @Step
   public NaviSideBar openSideMenu(String clusterName, MenuItem menuItem) {
     WebUtil.clickByActions(expandCluster(clusterName).parent()
-        .$x(String.format(sideMenuOptionElementLocator, menuItem.getNaviTitle())));
+        .find(By.linkText(menuItem.getNaviTitle())));
```

Comment:
This selector seems to be more stable as the previous selector was failing in my environment